GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -130,6 +133,34 @@
     public final GVRTextureParameters DEFAULT_TEXTURE_PARAMETERS = new GVRTextureParameters(
             this);
 
+    /* *
+     * Store the mapping of integers with wrap modes   
+     */
+    public HashMap<Integer,TextureWrapType>WrapmodeMap=
```

Comment:
Deriving an anonymous class just for this? What's wrong with having a static array with 3 TextureWrapType elements and looking up by index? It will be more readable also.